### ISSUE TITLE
Increase max connection limit for NDB instances

### ISSUE THREAD

**Issue by blancharda** [defenseunicorns/uds-bundle-software-factory-nutanix#76]
NDB instances appear to have a 100 connection limit by default. Gitlab is producing over 90 with basic/minimal usage. We should check to see if this is a parameter that can be modified via terraform

**Comment by jacobbmay** [comment]
This is something I ran into while doing GitLab performance testing, but it hasn't been an issue for us when just installing GitLab and not really doing anything with it.

The connection limit is something I modified while tuning postgres for GitLab performance. Unfortunately NDB only exposes some postgres settings and anything it doesn't expose needs configured directly in the postgres DB. The connection limit is one of the settings that it does expose, but like half of the settings I updated for performance tuning aren't available through NDB so I haven't found a good clean way to do it all via terraform yet that will leave it customizable since most of the performance tuning settings change based on DB resources available. 

Also NDB only lets you configure settings at deploy time using a parameter profile. Once the DB is deployed, settings changes need to be made directly on the instance so even if it were done via terraform initially, future changes would need to be made directly on the instance.
